charts/cnwan-operator/values.yaml

```diff
@@ -0,0 +1,127 @@
+image:
+  # image.registry -- The registry where to get the container image.
+  registry: ghcr.io
+  # image.repository -- The repository where to get the container image inside
+  # the registry specified above.
+  repository: cloudnativesdwan/cnwan-operator
+  # image.tag -- The tag for the container image.
+  # @default -- The most recent version will be used.
+  tag: v0.5.0
```

Comment:
We should change this `latest`? At least that's what the `@default` hints at

Reply:
Resolving as already discussed on WebEx.